src/main/java/com/lms/learning_management_system/controller/UserController.java

```diff
@@ -0,0 +1,63 @@
+package com.lms.learning_management_system.controller;
+
+import com.lms.learning_management_system.dto.UserDTO;
+import com.lms.learning_management_system.entities.service.UserService;
+import com.lms.learning_management_system.exception.UserNotFoundException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+
+    private final UserService userService;
+
+    @Autowired
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @PostMapping
+    public UserDTO createUser(@RequestBody UserDTO userDTO) {
+        return userService.createUser(userDTO);
+    }
```

Comment:
You can catch up @ResponseStatus(HttpStatus.CREATED)
Do this for all controller methods with the appropriate status

Reply:
done